### DISCUSSION TITLE
Umi2.X向下兼容问题

### DISCUSSION THREAD

**Discussion by 010101010jasgjsgdga** [umijs/umi#8143] [Show and tell]
Umi2.X 向下兼容
对于低版本浏览器不兼容，用官方提供的 “target：{}” 方式并不起作用，也尝试了各种垫片均无法生效。
（react+ant-design-pro/umi2.x  想兼容chrome44版本）

**Comment by qiutian00** [comment]
看这个 [umi2地址](https://v2.umijs.org/config/#targets)  [umi3地址](https://umijs.org/config#targets)

**Reply by 010101010jasgjsgdga** [reply]
尝试了 并没有效果
